<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NomismaSol" failures="1" tests="1" name="15. LinkedWithAccountsTest(failed)" time="5.768" errors="0" timestamp="8 Jun 2018 02:50:49 GMT">
  <testcase classname="test.java.com.nomisma.scriptspayroll.LinkedWithAccountsTest" name="@AfterMethod postCondition" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at test.java.com.nomisma.generic.BaseLib.postCondition(BaseLib.java:141)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod postCondition -->
  <testcase classname="test.java.com.nomisma.scriptspayroll.LinkedWithAccountsTest" name="lnkWithAccountsTest" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at test.java.com.nomisma.scriptspayroll.LinkedWithAccountsTest.lnkWithAccountsTest(LinkedWithAccountsTest.java:25)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- lnkWithAccountsTest -->
</testsuite> <!-- 15. LinkedWithAccountsTest(failed) -->
